docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
     build:
       context: depobs/
     container_name: dependency-observatory-api
-    image: dependency-observatory-api
+    image: gguthemoz/dependency-observatory
```

Comment:
This will build and tag it locally aas gguthemoz/dependency-observatory, so `docker-compose push api` publishes it there.